picturae_csv_create.py

```diff
@@ -321,18 +322,16 @@ def barcode_has_record(self):
                 self.record_full.loc[index, 'barcode_present'] = True
 
     def image_has_record(self):
-        """checks if image name/barcode already in attachments table"""
+        """checks if image name/barcode already in image_db"""
         self.record_full['image_present'] = None
         for index, row in self.record_full.iterrows():
             file_name = os.path.basename(row['image_path'])
             file_name = file_name.lower()
             # file_name = file_name.rsplit(".", 1)[0]
-            sql = f'''select origFilename from attachment
-                      where origFilename = "{file_name}";'''
-            db_name = self.specify_db_connection.get_one_record(sql)
-            if db_name is None:
+            imported = self.image_client.check_image_db_if_filename_imported(collection="Botany",
```

Comment:
checking image_db with api instead of attachments table ,to avoid double importing images without attachment table records like type or redacted images. (creating accidental 409 errors)